.appveyor.yml

```diff
@@ -8,11 +8,14 @@ environment:
     - nodejs_version: "10"
 
 cache:
+  - '%APPDATA%\npm-cache -> package.json' # Only load this from cache if package.json is unchanged
   - '%LOCALAPPDATA%\Yarn'
   - '%USERPROFILE%\.electron'
 
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
+  - cmd: IF "%nodejs_version%"=="6" npm install -g npm@6
+  - npm --version
   - npm install yarn -g
```

Comment:
If we care, we can take this out now since [AppVeyor ships with yarn 1.7.0](https://www.appveyor.com/updates/2018/06/08/).